app/models/user.rb

```diff
@@ -1,5 +1,10 @@
 # Top-level documentation comment for User module
+
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] As stated in the Project requirement Kindly make sure that **The application asks for confirmation of email.** Kindly make sure that `:confirmable` is enabled in the user model (`user.rb` file).
	
```
devise :database_authenticatable, :registerable,
				:recoverable, :rememberable, :validatable, :confirmable
```
 

- When I try to Login with combination of email and password the application doesn't ask me for the confirmation of email so Kindly Fix the issue.
- The application sign in without ask me for confirmation of email as shown with the below screenshot
![image](https://github.com/yayaismaail/blog/assets/93598189/36be4594-2ac3-49cb-8cfa-7f4d24cdc3a2)

Reply:
Hello @Abenezer-Tilahun, Thank you for your thorough feedback